docs/src/usage/nodejs.md

```diff
@@ -152,3 +152,46 @@ const acceptedPolicy = auth.authorize();
 const results = auth.query(rule`u($id) <- user($id)`);
 console.log(results.map(fact => fact.toString()));
 ```
+
+## Using biscuit with [express](https://expressjs.com)
+
+[Express](https://expressjs.com) is a popular web framework for NodeJS. `biscuit-wasm` provides support for express through a dedicated [middleware](https://expressjs.com/en/guide/using-middleware.html).
+
+Here is a minimal example of an application exposing a single `/protected/:dog` endpoint, and requiring a token with a corresponding `right()` fact.
+
+Calling `middleware` with an options object provides a middleware builder, which takes either an authorizer or a function building an authorizer from a request, and returns an actual middleware. This middleware generates an authorizer from the options and the builder, runs the authorization process and either aborts the request if authorization fails or passes control over to the endpoint handler if authorization succeeds.
```

Comment:
this whole bit is quite terrible honestly. not sure how to improve it, honestly.